pstoremgr/pstoremgr.go

```diff
@@ -312,14 +312,14 @@ func (pm *Manager) Bootstrap(count int) []peer.ID {
 			continue
 		}
 
-		logger.Debugf("connecting to %s", pinfo.ID)
```

Comment:
No reason to change these and below. They were more verbose in the past and I had to tune it down because it seriously spams the logs on big clusters (and not being able to connect some peer during bootstrap is not a big deal).